Controller/RoleController.php

```diff
@@ -27,18 +26,17 @@ class RoleController extends Controller
      * @param $type
      * @param int $id
      *
-     * @return array
-     * @Sensio\Bundle\FrameworkExtraBundle\Configuration\Route(
+     * @Config\Route(
```

Comment:
obecne `Sensio\Bundle\FrameworkExtraBundle\Configuration\Route` je deprecated, slo by misto toho pouzit `Symfony\Component\Routing\Annotation\Route` tak jako je to jinde aby to v celem projektu bylo sjednoceno?

Reply:
opraveno.